pkg/validations/createvalidations/cluster.go

```diff
@@ -29,12 +27,3 @@ func ValidateManagementCluster(ctx context.Context, k validations.KubectlClient,
 	}
 	return k.ValidateEKSAClustersCRD(ctx, cluster)
 }
-
-func ValidateTaintsSupport(ctx context.Context, clusterSpec *cluster.Spec) error {
```

Comment:
Why are we removing this?

Reply:
Since we have a similar method in upgrade, we will just import that and use it, instead of defining same method twice.

Reply:
oh cool, I like that
What do you think about moving it to a shared package? instead of importing `upgradevalidations` from `createvalidations`